### ISSUE TITLE
Performance issues with the Event Calendar

### ISSUE THREAD

**Issue by csaborio001** [soflyy/breakdance-bugs#1018]
Every aspect of Breakdance's performance is negatively affected when the events calendar plugin is installed.

### Steps to repro

1. Install Breakdance
2. Install The Events Calendar (6.2.8.2)
3. Create a few templates in breakdance and access the Templates section
4. It will take about 12 seconds to load the page ([video here](https://1drv.ms/v/s!AuYDDdfI2CCigYPPfZaRiJHCpuOaQdc)
5. Uninstall The Events Calendar plugin
6. Access the Templates section and reload
7. The templates will load almost instantly ([video here](https://1drv.ms/v/s!AuYDDdfI2CCigYPPfg9u3Rj73qEAqFY))

There are no errors show in the console for either scenario.  The performance issue extend to the Breakdance editor where loading any sort of template takes about 10 seconds to load.

In the front end, everything works as expected, it's just Breakdance in the backend that really takes a hit in terms of performance.

**Comment by tydiamond11** [comment]
I have noticed this same issue along with the Events Tickets, causing issues with plugin compatibility only noticed in the editor area of breakdance.

**Comment by csaborio001** [comment]
Yes, I noticed that as well, I don't think either company wants to address this issue.
